build/validate.sh

```diff
@@ -1,14 +1,17 @@
 #!/bin/bash
-echo -e "\nChecking go fmt bosun.org/..."
-GOFMTOUT=$(go fmt -s bosun.org/...);
-if [ "$GOFMTOUT" != '' ]; then 
-    echo "The following files need gofmt -s:";
-    echo "$GOFMTOUT";
-    exit 1;
+echo -e "\nChecking gofmt -l -s -w for all folders that don't start with . or _"
+cd $GOPATH/src/bosun.org
+GOFMTRESULT=0
+GOFMTOUT=$(find . -maxdepth 1 -type d -iregex './[^._].*' -exec gofmt -l -s -w {} +);
```

Comment:
@mjibson I fixed the -l, added the DIRS variable, and force pushed, but `go vet $DIRS` is complaining about some folders. Should I exclude those when calling go vet?

```
[root@ny-gbraylx01 bosun.org]# go vet $DIRS
can't load package: package bosun.org/cmd: no buildable Go source files in /home/gbray/go/src/bosun.org/cmd
can't load package: package bosun.org/plugins: no buildable Go source files in /home/gbray/go/src/bosun.org/plugins
can't load package: package bosun.org/build: no buildable Go source files in /home/gbray/go/src/bosun.org/build
```